css/mypage.css

```diff
@@ -182,6 +182,10 @@ margin: auto;
   overflow-y: auto;
 }
 
+#hamburger.hamburger-open + #burger-menu {
```

Comment:
"Element mit ID 'hamburger' mit class 'hamburger-open' direkt gefolgt von dem Element mit ID 'burger-menu'" (das Styling wird dann auf das letzte Element des Selektors angewendet, also "#burger-menu")

Reply:
nach zwei Sekunden mehr drüber nachdenken: Wäre einfacher gewesen, die css Klassen an das burger-menu zu hängen, so dass am Ende der Selector einfach nur `#burger-menu.burger-menu-open` gewesen wäre. (entsprechendes muss man dann auch in der mypage.js das anpassen...)